modules/users/server/config/users.server.config.js

```diff
@@ -11,7 +11,7 @@ var passport = require('passport'),
 /**
  * Module init function
  */
-module.exports = function (app, db) {
+module.exports = function (app) {
```

Comment:
this could potentially break vertical modules that rely on it.
even with the changes in this PR for `config/lib/mongoose.js` how would one be able to access the db instance?

the other alternative I can think of is if we set app.local.db or something like that which would expose the db connection where app is too.

WDYT?